pkg/cluster/nodes/create.go

```diff
@@ -145,6 +145,9 @@ func createNode(name, image, clusterLabel, role string, mounts []cri.Mount, port
 		// runtime temporary storage
 		"--tmpfs", "/tmp", // various things depend on working /tmp
 		"--tmpfs", "/run", // systemd wants a writable /run
+		// FHS State information. Persistent data modified by programs as they run,
+		// e.g., databases, packaging system metadata, etc.
+		"--tmpfs", "/var/lib",
```

Comment:
You shouldn't make all of var a tmpfs.

Reply:
This will consume inordinate memory for one.

Reply:
I see the mistake now 😓